src/Microsoft.DotNet.Interactive.Formatting/JsonFormatter.cs

```diff
@@ -31,7 +30,7 @@ public static ITypeFormatter GetPreferredFormatterFor(Type type)
 
         internal static ITypeFormatter[] DefaultFormatters { get; } = DefaultJsonFormatterSet.DefaultFormatters;
 
-        public static JsonSerializerSettings SerializerSettings { get; }
+        public static JsonSerializerOptions SerializerSettings { get; }
```

Comment:
Should probably rename the property to match the type.

Reply:
done